pkg/kubelet/kubelet_pods.go

```diff
@@ -111,6 +111,23 @@ func (kl *Kubelet) makeDevices(pod *v1.Pod, container *v1.Container) ([]kubecont
 	return devices, nil
 }
 
+func makeAbsolutePath(goos, path string) string {
```

Comment:
Should this just be in kubelet_windows.go and kubelet_linux.go files?

Reply:
I like it here better, since that way it will run in the unit tests on either platform.